templates/kubernetes/index.html

```diff
@@ -10,20 +10,38 @@
 <section class="p-strip--suru-topped is-bordered">
   <div class="row u-equal-height u-vertically-center">
     <div class="col-7">
-      <h1>Enterprise Kubernetes for multi-cloud operations</h1>
-      <p class="p-heading--4">Container orchestration streamlined from cloud to edge</p>
+      <h1>Canonical Kubernetes for multi-cloud operations</h1>
+      <p class="p-heading--4">From the operating system to a full cloud native stack</p>
       <p>
         Ubuntu is the reference platform for Kubernetes on all major public clouds.<br>
         Canonical Kubernetes is built on Ubuntu and combines security with optimal price-performance.<br>
-        Our cloud-native automation framework ensures developer productivity and business innovation.
+        Canonical is a Certified Kubernetes Service Provider, your trusted advisor for a successful cloud native strategy.<br>
+        Our cloud native automation framework ensures developer productivity and business innovation.
       </p>
```

Comment:
Would it be better to wrap each each sentence with `<p>` instead of using `<br>` What do you think?
![image](https://user-images.githubusercontent.com/57550290/163334585-0f3744d2-89b2-4d8e-9d24-b23c53a0ce75.png)